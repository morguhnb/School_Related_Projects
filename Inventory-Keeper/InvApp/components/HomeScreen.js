import { View, Text, StyleSheet, TextInput, TouchableOpacity,} from 'react-native';

const HomeScreen = props => {
    return(
    
    <View style={styles.header}>
      <Text style={styles.buttonText}>Inventory Keeper</Text>



      <TouchableOpacity style={styles.button} onPress={() => props.navigation.navigate('AddItem')}>
      <Text style={styles.buttonText}>Add Item</Text>
      </TouchableOpacity>

      <TouchableOpacity style={styles.button} onPress={() => props.navigation.navigate('ListItem')}>
      <Text style={styles.buttonText}>Item List</Text>
      </TouchableOpacity>

      <TouchableOpacity style={styles.button} onPress={() => props.navigation.navigate('WelcomePage')}>
      <Text style={styles.buttonText}> Go back </Text>
      </TouchableOpacity>


    </View>
    
  )
}

const styles = StyleSheet.create({
  input: {
      height: 60,
      padding: 8,
      fontSize: 16
      
  },
  button: {
    backgroundColor: '#c2bad8',
    padding: 30,
    margin: 5
  },
  buttonText: {
      color: "darkslateblue",
      fontSize: 20,
      textAlign: 'center'
  }

})
export default HomeScreen;