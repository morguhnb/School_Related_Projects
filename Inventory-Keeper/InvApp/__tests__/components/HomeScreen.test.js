import React from "react";
import renderer from "react-test-renderer";

import HomeScreen from "../../components/HomeScreen";

describe("<HomeScreen />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<HomeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
