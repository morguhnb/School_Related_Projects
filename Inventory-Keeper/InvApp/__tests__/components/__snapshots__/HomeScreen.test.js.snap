// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "darkslateblue",
        "fontSize": 20,
        "textAlign": "center",
      }
    }
  >
    Inventory Keeper
  </Text>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#c2bad8",
        "margin": 5,
        "opacity": 1,
        "padding": 30,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "darkslateblue",
          "fontSize": 20,
          "textAlign": "center",
        }
      }
    >
      Add Item
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#c2bad8",
        "margin": 5,
        "opacity": 1,
        "padding": 30,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "darkslateblue",
          "fontSize": 20,
          "textAlign": "center",
        }
      }
    >
      Item List
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#c2bad8",
        "margin": 5,
        "opacity": 1,
        "padding": 30,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "darkslateblue",
          "fontSize": 20,
          "textAlign": "center",
        }
      }
    >
       Go back 
    </Text>
  </View>
</View>
`;
